pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -2550,8 +2550,8 @@ self: super: {
 
   # attoparsec bump is on v2 branch, but not released yet
   irc-core = assert super.irc-core.version == "2.10"; doJailbreak super.irc-core;
-  glirc = assert super.irc-core.version == "2.10"; doJailbreak super.glirc;
-  hookup = assert super.irc-core.version == "2.10"; doJailbreak super.hookup;
+  # glirc 2.39 is broken, downgraded to 2.38 to unbreak
+  glirc = assert super.irc-core.version == "2.10"; doJailbreak (markUnbroken super.glirc);
```

Comment:
`markUnbroken` can be dropped now. Not sure if the jailbreaks are still necessary (probably?).

Reply:
Right! Jailbreaking is still necessary for attoparsec bounds present in glirc 2.38 and irc-core.